packages/react-atlas-docs/src/components/dropdownList/DropdownList.js

```diff
@@ -0,0 +1,13 @@
+import React, { PropTypes } from "react";
+import * as Atlas from 'react-atlas';
+
+export default class DropdownList extends React.Component {
+  constructor(props){
+    super(props)
+  }
+  render() {
+    return (
+      <Atlas.Button {...this.props}></Atlas.Button>
```

Comment:
Dropdown isn't finished. Thus PR is only for avatar, card, and button. The rest of the components have copied components from Button